### ISSUE TITLE
Post softmax instead of pre softmax

### ISSUE THREAD

**Issue by canbakiskan** [IBM/ZOO-Attack#5]
~~Hello,~~

~~I see that this repo is pretty old by now but I'll give this issue a shot. I was going over the code and I have a question:~~

~~In [https://github.com/IBM/ZOO-Attack/blob/93f09bd/l2_attack_black.py](https://github.com/IBM/ZOO-Attack/blob/93f09bd/l2_attack_black.py), shouldn't line 255 be~~

```
if use_log:
    self.output = tf.nn.softmax(model.predict(self.newimg),1)
else: 
    self.output = model.predict(self.newimg)
```

~~instead of just~~

```
self.output = model.predict(self.newimg)
```

**Comment by pinyuchen** [comment]
Hello @canbakiskan 

I didn't find the codes you are referring to in "l2_attack_black.py". Could you provide a link?

**Comment by canbakiskan** [comment]
I'm sorry it was my mistake. I didn't see the softmax addition in model's prediction. You can delete this issue, sorry to bother.

**Comment by pinyuchen** [comment]
No worries, just about to say the same :)
